R/stan_module.R

```diff
@@ -76,6 +77,11 @@ setMethod(
             msg = "`Priors` and `inits` must be lists"
         )
 
+        validate_that(
+            length(inits) == length(priors),
+            msg = "Lengths of inits and priors differ"
+        )
+
```

Comment:
How come you've used `validate_that` instead of `assert_that`, don't we want to throw an error if the condition isn't met ?

Reply:
Because of the case that the user defines the prior list (which is essential for the model built) but not the initial values. In that case, the lengths of the two lists are going to be different.